### ISSUE TITLE
Mission files are incompatible with a game progress saves

### ISSUE THREAD

**Issue by Varrkan82** [tbox1911/Liberation-RX#1]
Hi.
Last few attempts to merge changes from your branch to my localized one was unsuccessful. I'm assume - there are some changes in mission.sqm file which are bricking the saved FOB and Base positions. After merging changes today and re-building the mission I'm spawned on a southern ilend of an Altis map. Could it be fixed without a game progress reset?

**Comment by tbox1911** [comment]
Hi,
Yes, I have added some change to mission sqm (welcome flag, new marker, and so)
the save manager has been also upgraded.

You can try to backup your save and test with a save game reset to check if it's work.
take a look to your RPT file to check for error

keep me in touch

**Comment by Varrkan82** [comment]
> keep me in touch

Thanks for fast reply.
I'm figured out it is not a problem of a changing Saving system. At least - not only this. Looks like I've broke something in a spawning players. Although all looks well in my modifications - sometimes player could be spawned, but usually not. I see only a map demonstration (flying under the island). I've tried different maps and have the same. :(

Checked my translation and modifications twice with a Meld diff-tool and manually checked all different files - all looks fine.

This issue could be closed I think
